step.yml

```diff
@@ -230,6 +230,33 @@ inputs:
         - "Japanese"
       is_expand: true
       category: "detail"
+  - device_region: "Default"
```

Comment:
I'm not sure, but maybe you have to add "Default" also in `value_options`.
You can confirm the actual behavior by the procedure noted [here](https://github.com/NozomiIto/magic-pod/blob/develop/docs/development_tips/bitrise_step.md#check-ui-on-bitriseio)